<ui version="4.0" >
 <class>SANSPlotSpecial</class>
 <widget class="QFrame" name="SANSPlotSpecial" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Display</string>
  </property>
  <property name="frameShape" >
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow" >
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QGroupBox" name="gbPlotOptions" >
       <property name="title" >
        <string>Plot Options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <layout class="QHBoxLayout" name="layoutInput" >
          <item>
           <widget class="QLabel" name="lbInput" >
            <property name="text" >
             <string>Workspace</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MantidQt::MantidWidgets::WorkspaceSelector" name="wsInput" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutBackground" >
          <item>
           <widget class="QLabel" name="lbBackground" >
            <property name="text" >
             <string>Background</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbBackground" >
            <item>
             <property name="text" >
              <string>Value</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Workspace</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="swBackground" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex" >
             <number>0</number>
            </property>
            <widget class="QWidget" name="page" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3" >
              <property name="spacing" >
               <number>0</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QDoubleSpinBox" name="dsBackground" />
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4" >
              <property name="spacing" >
               <number>0</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="MantidQt::MantidWidgets::WorkspaceSelector" name="wsBackground" />
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" >
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Plot Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbPlotType" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutXAxis" >
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>X Axis:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutYAxis" >
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Y Axis:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lbPlotOptionsError" >
          <property name="styleSheet" >
           <string notr="true" >color:rgb(255, 0, 0)</string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" >
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>148</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbPlot" >
            <property name="text" >
             <string>Plot</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" >
          <item>
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Intercept:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbInterceptValue" >
            <property name="styleSheet" >
             <string notr="true" >color:rgb(0, 0, 255)</string>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" >
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Gradient:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbGradientValue" >
            <property name="styleSheet" >
             <string notr="true" >color: rgb(0,0,255)</string>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" >
          <item>
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Chi Squared:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbChiSqrdValue" >
            <property name="styleSheet" >
             <string notr="true" >color: rgb(0,0,255)</string>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QwtPlot" name="plotWindow" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QPushButton" name="pbHelp" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::WorkspaceSelector</class>
   <extends>QComboBox</extends>
   <header>MantidQtMantidWidgets/WorkspaceSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
